### ISSUE TITLE
pyTempNet/Log.py" => from enum import IntEnum ImportError: No module named enum

### ISSUE THREAD

**Issue by abitofalchemy** [IngoScholtes/pyTempNets#14]
I'm trying out pyTempNets, impressive tutorial btw, but I've run into the following error:
`pyTempNet/Log.py", line 9, in <module>
    from enum import IntEnum
ImportError: No module named enum`

Are pyTempNet pre-req's that python3 be used?

**Comment by IngoScholtes** [comment]
Hi

thanks for your comment. Indeed, it seems that the Log component creates a dependency to python 3.4, while the rest of the module has been developed to work both on python 2.7 and python 3.4. 

I don't see an immediate fix for this issue, but we'll have a closer look on how we can avoid this in the next release.

In the meantime: If you still want to work with the module, I recommend setting up an Anaconda environment for python 3.5 (https://www.continuum.io/downloads). This should leave your current python setup untouched, and it will also allow you to switch back and forth between python 2.7 and 3.5. 

Thanks for your patience, 

Ingo

**Comment by abitofalchemy** [comment]
The above was tried on a cluster, for which I have no root access. I can't try Anaconda on that system.  I'm trying to install it now on one of my own machines, but keep running into `UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 32: ordinal not in range(128)`  I don't know if the error is related to missing libs & compilers.  For example, after an N # of tries, my system is still missing gfortran, so I wonder if after all those are properly installed ---will it still fail at the mentioned error?

**Comment by romanc** [comment]
Please be patient. We are working on a solution to the enum issue building on its backport [`enum34`](https://pypi.python.org/pypi/enum34). This should then be compatible down to at least python 2.7.

**Comment by abitofalchemy** [comment]
Good news: on a newly minted linux box, after manually installing the dependencies I mentioned before, pip install command worked.  I now have-- a system with a working pyTempNets module.  I will work with this to play with some toy-networks! 

(if I should email you regarding the issue below --or continue to post it on this issue, let me know):
t1.exportTikzUnfolded('output/t1.tex') is returning an error: 

```
>>> t1.exportTikzUnfolded('output/t1.tex')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: TemporalNetwork instance has no attribute 'exportTikzUnfolded'
```

**Comment by romanc** [comment]
@abitofalchemy: Nice to hear, that the other installation is working. 

The compatibility issue will be resolved in  pull request #13 referenced below. It also hot-fixes some other minor issues.

For the error in Tikz-Output: The function `exportTikzUnfolded` has be moved to the `Visualizer` class. This is already corrected in the [online tutorial](https://www.sg.ethz.ch/team/people/ischoltes/research-insights/temporal-networks-demo/). Could you point us to the place you have encountered it?

Adapting names from the above example the new syntax would be 

``` python
import pyTempNet as tn
...
tn.Visualizer.exportTikzUnfolded( t1, "output/t1.tex" );
```

**Comment by abitofalchemy** [comment]
[found it here](http://www.ingoscholtes.net/research/insights/Temporal_Networks.html). Yes that fixed it, it's working!
